atc/api/configserver/save.go

```diff
@@ -69,19 +69,28 @@ func (s *Server) SaveConfig(w http.ResponseWriter, r *http.Request) {
 	}
 
 	pipelineName := rata.Param(r, "pipeline_name")
-	warning := atc.ValidateIdentifier(pipelineName, "pipeline")
+	warning, err := atc.ValidateIdentifier(pipelineName, "pipeline")
+	if err != nil {
+		session.Info("ignoring-invalid-config", lager.Data{"error": err.Error()})
```

Comment:
Maybe change this log message:

```suggestion
		session.Info("ignoring-invalid-pipeline-name", lager.Data{"error": err.Error()})
```

similarly for the team name